Counter/js.js

```diff
@@ -0,0 +1,40 @@
+let num = 0;
+
+function inc1() {
+  num += 1;
+  document.getElementById("num").innerHTML = num;
+}
+
+function inc2() {
+  num += 2;
+  document.getElementById("num").innerHTML = num;
+}
+
+function inc3() {
+  num += 3;
+  document.getElementById("num").innerHTML = num;
+}
+
+function dec1() {
+  num > 0 ? (num -= 1) : "";
+  document.getElementById("num").innerHTML = num;
+}
+
+function dec2() {
+  num === 1 ? (num -= 1) : "";
+  num > 0 ? (num -= 2) : "";
+
+  document.getElementById("num").innerHTML = num;
+}
+
+function dec3() {
+  num === 1 ? (num -= 1) : "";
+  num === 2 ? (num -= 2) : "";
+  num > 0 ? (num -= 3) : "";
+  document.getElementById("num").innerHTML = num;
+}
```

Comment:
```suggestion
const dec = (value) => {
  document.getElementById('num').innerHTML = num >= value ? (num -= value) : 0;
};
```

We may write a universal function for decrement too. To simplify everything we may just check if num is more than value and then do all calculations.